doc/subframes/subframes_tutorial.rst

```diff
@@ -24,15 +19,24 @@ so you can do things like this:
 
 In this animation, the robot moves the tip of the cylinder to different positions on the box.
 
+Getting Started
+---------------
+If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
+
+**Note:** This tutoral has made use of xterm and a simple prompter to help the user choose from each demo option.
+To install xterm please run the following command: ::
+
+   sudo apt-get install -y xterm
+
 Running The Demo
 ----------------
 After having completed the steps in `Getting Started <../getting_started/getting_started.html>`_, open two terminals. In the first terminal, execute this command to load up a panda, and wait for everything to finish loading: ::
 
-    roslaunch panda_moveit_config demo.launch
+   ros2 launch moveit_resources_panda_moveit_config demo.launch.py
 
 In the second terminal run the tutorial: ::
 
-    rosrun moveit_tutorials subframes_tutorial
+   ros2 launch moveit2_tutorials subframes_tutorial.launch.py
 
 In this terminal you should be able to enter numbers from 1-12 to send commands, and to see how the robot and the scene react.
```

Comment:
line 46 below should be `moveit2_tutorials`.

Reply:
Good catch, fixing it.